solr/core/src/java/org/apache/solr/search/facet/FacetContext.java

```diff
@@ -86,6 +87,14 @@ public FacetContext getParent() {
     return parent;
   }
 
+  public FacetContext nearestPermContext() {
```

Comment:
perm? or maybe nearestParentContext?

Reply:
Yeah, I can just simplify and call it `perm`. That SGTM.